server/utils/brew.js

```diff
@@ -0,0 +1,41 @@
+const yaml = require('js-yaml');
+
+module.exports = {
+	mergeBrewText : (brew, { style, metadata, fullMetadata })=>{
```

Comment:
Is there a particular reason this was split out into a separate file? I can see the benefit of Utils files but where this is a single function on its own that isn't used anywhere else, I don't think we need to split this out just yet.

Could we merge this back into `homebrew.api.js` for now to minimize the amount of movement in the code, and then in a future PR deal with any cleanup and refactoring?

Reply:
It's also used in `server.js` now in the download/source routes